convert/jenkinsjson/covert.go

```diff
@@ -281,28 +286,15 @@ func recursiveParseJsonToSteps(currentNode jenkinsjson.Node, steps *[]*harness.S
 	return nil, nil
 }
 
-func collectStepsWithID(currentNode jenkinsjson.Node, stepWithIDList *[]StepWithID, processedTools *ProcessedTools, variables map[string]string, timeout string) (*harness.CloneStage, *harness.Repository) {
+func collectStepsWithID(currentNode jenkinsjson.Node, stepWithIDList *[]StepWithID, processedTools *ProcessedTools, variables map[string]string, timeout string, dockerImage string) (*harness.CloneStage, *harness.Repository) {
 	var clone *harness.CloneStage
 	var repo *harness.Repository
 
-	if len(currentNode.ParameterMap) == 0 {
```

Comment:
Can you pull latest

Reply:
done

Reply:
still not correct it seems

Reply:
you need to pull across branches

Reply:
fixed now

Reply:
still not fixed. Maybe you need to do it manually